cypress/e2e/settings/thesauri.cy.ts

```diff
@@ -20,104 +25,212 @@ describe('Thesauri configuration', () => {
+    cy.contains('First Item')
+      .parentsUntil('tr')
+      .parent()
+      .eq(0)
+      .within(() => cy.get('input[type="checkbox"]').check());
+    cy.contains('button', 'Remove').click();
+    cy.contains('Are you sure you want to delete this item');
     cy.contains('button', 'Accept').click();
-    cy.contains('Thesauri deleted');
-    cy.get('tbody tr').should('have.length', 1);
+    saveThesaurus();
   });
 
-  it('should not delete when in use', () => {
-    cy.intercept('GET', 'api/templates').as('fetchtemplates');
-    cy.contains('span', 'Templates').click();
-    cy.contains('Edit').click();
-    cy.get('aside .list-group-item:nth-of-type(3) button').click();
-    cy.get('.metadataTemplate li:nth-of-type(3) .property-edit').click();
-    cy.get('select').eq(1).select('New Thesauri with groups edited');
-    cy.contains('Save').click();
-    cy.wait('@fetchtemplates');
+  it('should import items from csv', () => {
+    cy.contains('button', 'Import').click();
+    cy.get('input[type=file]').selectFile('./cypress/test_files/thesaurus-test.csv', {
+      force: true,
+    });
+    cy.contains('Thesauri updated.');
+    cy.contains('Dismiss').click();
+    cy.get('tbody').toMatchImageSnapshot();
+  });
+
+  it('should sort the items alphabetically', () => {
+    cy.contains('button', 'Sort').click();
+    cy.get('tbody').toMatchImageSnapshot();
+    changeLanguage('Español');
+    cy.contains('Colores');
+    cy.contains('button', 'Ordenar').click();
+    cy.get('tbody').toMatchImageSnapshot();
+  });
+
+  it('should cancel the changes', () => {
+    cy.contains('button', 'Cancelar').click();
+    cy.contains('button', 'Descartar cambios').click();
+  });
+
+  it('should use a value when creating an entity', () => {
+    cy.contains('a', 'Colección').click();
+    changeLanguage('English');
+    clickOnCreateEntity();
+    cy.get('[name="library.sidepanel.metadata.title"]').click();
+    cy.get('[name="library.sidepanel.metadata.title"]').type('País select', { delay: 0 });
+    cy.contains('select', 'Mecanismo').select('País');
+    cy.contains('.form-group.select select', 'Select...').select('Imported Blue');
+    cy.contains('button', 'Save').click();
+    cy.contains('Entity created');
+    cy.contains('.item-document', 'País select').click();
+    cy.contains('.metadata-name-select', 'Imported Colors: Imported Blue');
+    changeLanguage('Español');
+    cy.contains('.documentTypes-selector li', 'País').click();
+    cy.contains('.item-document', 'País select').click();
+    cy.contains('.metadata-name-select', 'Colores Importados: Azul Importado');
+  });
+
+  it('should update the values in the entities', () => {
+    changeLanguage('English');
+    cy.get('.only-desktop a[aria-label="Settings"]').click();
     cy.contains('span', 'Thesauri').click();
-    cy.get('tbody tr:nth-of-type(1) input').should('be.disabled');
+    cy.contains('New Thesaurus')
```

Comment:
I think you can do a `cy.contains('tr', 'New Thesaurus').contains('button', 'Edit').click();` here

Reply:
Certainly, much simpler!